docker-compose.dev.yml

```diff
@@ -0,0 +1,82 @@
+version: "3.11"
+services:
+  postgres:
+    container_name: postgres-triplane
+    image: postgres:14.6
+    hostname: postgres
+    ports:
+      - "5432:5432"
+      - "5433:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: triplane
+      POSTGRES_DB: postgres-triplane
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    restart: on-failure
+    networks:
+      - my_network
+
+  redis:
+    image: redis
+    container_name: redis
+    build:
+      context: ./backend/docker/redis
+      dockerfile: Dockerfile
+    privileged: true
+    env_file:
+      - ./backend/.env
+    restart: on-failure
+    ports:
+      - "6379:6379"
+    volumes:
+      - redis_data:/data:rw
+    healthcheck:
+      test: redis-cli ping
+      interval: 3s
+      timeout: 5s
+      retries: 5
+    networks:
+      - my_network
```

Comment:
```
redis:
    image: redis:latest
    ports:
      - "6379:6379"
    volumes:
      - redis-data:/data
     healthcheck:
      test: redis-cli ping
      interval: 3s
      timeout: 5s
      retries: 5
```
you dont need to create Dockerfile with redis

Reply:
yes, I need it, because Windows has some memory problems, but now I know how to do it, all I need to do in init.sh is:
redis-server --bind 0.0.0.0 --requirepass "${REDIS_PASSWORD}" --maxmemory 256mb --maxmemory-policy allkeys-lru --appendonly yes